### PR TITLE
New stations along route UI

### PR DESCRIPTION
**Pull request by Wouter125**
- [x] New UI
- [x] Moved route to EcoMovement data instead of OCM
- [x] Updated documentation

### CODE REVIEW

File: stations-along-route/index.js [hunk 1]
```diff
@@ -88,63 +86,43 @@ let routeData;
...
+ * @param { object } data - All available route data
  */
-const displayRouteData = data => {
-  document.getElementById('loader').remove();
-  document.getElementById('showAlternativeStationsSwitcher').removeAttribute('disabled');
-
-  document.querySelector('.tags').style.display = 'flex';
+const renderRouteHeader = data => {
+  const routeDistance = data.distance ? `${(data.distance / 1000).toFixed(0)} km` : 'Unknown';
+  const routeStops = `${data.charges ?? 0} stops`;
```
Comment (lsmienk) [inline stations-along-route/index.js:101]: I don't think you are asking for charges in the query

File: stations-along-route/README.md [hunk 2]
```diff
@@ -22,23 +22,23 @@ We use our Playground environment for this example. It means that only part of o
 To build a route, you will need a car (the associated consumption model of a vehicle will be applied to the routing algorithm), station database, origin, and a destination.
 
 For this example, we use **Tesla model S**, **Amsterdam** as an origin, and **Osnabrück** as a destination point.
-Chargetrip operates an extensive database of EVs, each with their specific consumption models. You can find more information about our database and available queries by checking [Chargetrip API documentation](https://docs.chargetrip.com/#cars).
+Chargetrip operates an extensive database of EVs, each with their specific consumption models. You can find more information about our database and available queries by checking [Chargetrip API documentation](https://developers.chargetrip.com/API-Reference/Cars/query-cars).
```
Comment (lsmienk) [inline stations-along-route/README.md:25]: the route has changed, lets add the new origin/destination here as well.

**Review by lsmienk — CHANGES_REQUESTED** [review]
Found two small things. Everything els looks great.

**Comment by Wouter125** [comment]
If you good, we can merge this one as well.

**Review by lsmienk — APPROVED** [review]

**Pull request merged**